flink-runtime/src/main/java/org/apache/flink/runtime/rpc/RpcEndpoint.java

```diff
@@ -127,7 +143,7 @@ public final void start() {
 	}
 
 	/**
-	 * User overridable callback.
+	 * Starts the rpc endpoint.
```

Comment:
This JavaDoc is misleading because the user might think that he needs to call this method to start the `RpcEndpoint`. Better to say `Internal method which is called by the RpcService implementation to start the RpcEndpoint` or so.